.travis.yml

```
@@ -10,7 +10,7 @@ git:
 
 jobs:
   include:
-  - env: CMD="make validate"
+  - env: CMD="make validate" DEPLOY=true
```

```suggestion
  - env: CMD="make validate dapper-image" DEPLOY=true
```

? or do we call the dapper-image part from validate? I can't remember, may be it was part of "ci"

---

you're right, I forgot to add it to the Makefile, thanks